packages/lesswrong/server/manualMigrations/migrationUtils.ts

```diff
@@ -65,13 +66,31 @@ export async function runMigration(name: string)
     started: new Date(),
   });
   
+  const db = getSqlClient();
+
+  // We can't assume that certain postgres functions exist because we may not have run the appropriate migration
+  // This wraapper runs the function and ignores if it's not defined yet
+  const safeRun = async (fn) => {
+    if(!db) return;
```

Comment:
This seems bad. If the db hasn't been defined yet, throw.

Reply:
Won't that break for less wrong? I guess I don't really understand how the migration architecture is different for them

Reply:
(I don't know the right sol'n)